tests/run_examples.py

```diff
@@ -171,6 +171,13 @@ def check_library(rule):
 
 req_os = defaultdict(lambda: [])
 
+skiped_files = ['__init__.py',
+                'spmd_utils.py',
+                'log_reg_model_builder.py',
+                'n_jobs.py',
```

Comment:
but we would like to run those examples to validate that they are work

Reply:
> but we would like to run those examples to validate that they are work

Most of these files are not the examples, but some helper functions. For this reason we shouldn't run it as examples. The only exception is `log_reg_model_builder.py`. For some unknown reason this file didn't have any suffix, for this reason, it wasn't launching before at all. So we have two independent tasks here:
1. Remove `_batch` suffix for more clear naming (this PR).
2. Investigate the problem with `log_reg_model_builder.py` and prepare a fix for this problem (some other PR). After the problem will be fixed, we can remove this file from the list of skipping examples.

This plan looks reasonable from my point of view, what is your opinion?

Reply:
@razdoburdin I think as an option you can move non-example, helpers scripts into utils module `examples/utils`.